Module01/Lexer.py

```diff
@@ -1,288 +1 @@
-"""
-Project01 - CS5400
-Group Members: Daniel Hayes and Sarah Crane
-Semptember 18, 2024
-"""
-class Lexer:
-    """
-    Class Lexer
-    Processes input strings and produces tokens
-    """
-    def __init__(self, text):
-        """
-        Initializes the lexer with the input text.
-        Sets the initial position and current character.
-        """
-        self.text = text
-        self.charlist = list(text)  # Creates list of chars out of String input
-        self.current_char = self.charlist[0]    # Stores current char
-        self.position = 0       # Stores int position in char list
-        self.current_token = "" # Stores String of current token
-        self.period_counter = 0 # Counts periods in numbers (cannot have more than 1)
-        self.is_alpha = False   # Currently reading variable boolean - is True if going through variable
-        self.is_number = False  # Currently reading numerical boolean - is True is going through number
-        self.last_token_type = None # Stores previous token type for potential error handling
```

Comment:
Not used anywhere in the file. In the future, you would probably want to use an array of all previous tokens